src/Formatter/IntlMoneyFormatter.php

```diff
@@ -16,13 +16,19 @@
      * @var \NumberFormatter
      */
     private $formatter;
+    /**
+     * @var int|null
```

Comment:
It should be either `null` (optional) or required in the constructor. We could also make a named constructor for this. So then you have two options.

```php
$formatter = new IntlMoneyFormatter(new \NumberFormatter());
// or
$formatter = IntlMoneyFormatter::withSubunits(new \NumberFormatter(), 2);
```

The advantage is you can eliminate `null` from the constructor, which is a good thing because you want to avoid null as much as you can.